modules/kubernetes/eks-core/namespace.tf

```diff
@@ -25,36 +34,27 @@ resource "kubernetes_namespace" "tenant" {
   }
 }
 
-resource "kubernetes_service_account" "tenant" {
+resource "kubernetes_limit_range" "tenant" {
   for_each = { for ns in var.namespaces : ns.name => ns }
 
   metadata {
-    name      = each.value.name
-    namespace = kubernetes_namespace.service_accounts.metadata[0].name
+    name      = "default"
+    namespace = kubernetes_namespace.tenant[each.key].metadata[0].name
   }
-}
 
-#resource "kubernetes_limit_range" "tenant" {
-#  for_each = { for ns in var.namespaces : ns.name => ns }
-#
-#  metadata {
-#    name      = "default"
-#    namespace = each.key
-#  }
-#
-#  spec {
-#    limit {
-#      type = "Container"
-#      default_request = {
-#        cpu    = var.kubernetes_default_limit_range.default_request.cpu
-#        memory = var.kubernetes_default_limit_range.default_request.memory
-#      }
-#      default = {
-#        memory = var.kubernetes_default_limit_range.default.memory
-#      }
-#    }
-#  }
-#}
```

Comment:
Not really sure why this was commented out. @NissesSenap  do you have any insight into this?

Reply:
Don't know, my guess is that I wanted to keep everything as simple as possible initially and then I probably forgot to add it again.